src/main/java/org/opentosca/csarrepo/service/DownloadCsarFileService.java

```diff
@@ -65,4 +80,34 @@ public DownloadCsarFileObject getResult() {
 
 		return this.downloadCsarFileObject;
 	}
+
+	/**
+	 * Duplicates a given file
+	 * 
+	 * @param file
+	 * @return
+	 * @throws IOException
+	 */
+	private File duplicateFile(File file) throws IOException {
+		File duplicatedFile = File.createTempFile("abc", "zip");
```

Comment:
I'm not sure how good this one does behave on concurrent access - i can just guess BAD.

Think it would be better to create a tempFile with a random name (even if that is polluting the temp folder)